project/Octoparts.scala

```diff
@@ -20,7 +20,7 @@ object Octoparts extends Build {
   // -------------------------------------------------------
   lazy val app = Project(id = "octoparts", base = file("."), settings = playAppSettings)
     .enablePlugins(PlayScala)
-    .dependsOn(models, authHandlerApi, playJsonFormats)
+    .dependsOn(models, authHandlerApi, playJsonFormats, scalaWsClient % "test->compile")
```

Comment:
Yeah, this is a cryptic addition that adds a test-mode dependency in the current project definition (root app in this case); in particular the ScalaWsClient project is to be in a compiled state.

The reason I did this was to allow us to integration test the root project (Octoparts app) using the ScalaWsClient.